kvm-unit-tests.yaml

```diff
@@ -12,6 +12,7 @@ globals:
 projects:
 - name: LKFT-kvm-unit-tests
   projects: &projects_all
+    - lkft/linux-next-master
     - lkft/linux-next-oe
```

Comment:
To make this change, we will want to change the project slug - so please remove the old name (linux-next-oe) in this PR.

Reply:
Done.